ui/src/app/views/workspace/component.html

```diff
@@ -1,10 +1,16 @@
 <div *ngIf="!notFound else notFoundMessage">
   <div *ngIf="!workspaceLoading; else workspacePageLoading" style="display: flex;">
     <h2>{{workspace.name}}</h2>
-    <span *ngIf="hasPermission()">
-      <clr-icon class="editing-icon is-solid" shape="pencil" (click)="edit()"></clr-icon>
-    </span>
-    <clr-icon class="sharing-icon is-solid" shape="users" (click)="share()"></clr-icon>
+    <div class="icon-area">
+      <clr-icon *ngIf="hasWritePermission()"
```

Comment:
This may work better as a [button group](https://vmware.github.io/clarity/documentation/button-group) (or even better, a button toolbar, but that concept isn't in Clarity).